src/api-services/user.service.js

```diff
@@ -20,16 +20,17 @@ export default {
     return Axios.post(RESOURCE_NAME.concat('/login'), userData)
   },
   confirmEmail (id, token) {
-    return Axios.patch(RESOURCE_NAME.concat(`/emailVerification/${id}`), { token }, {
-      headers: {
-        accept: 'application/json'
-      }
-    })
+    return Axios.patch(RESOURCE_NAME.concat(`/emailVerification/${id}`), token)
+  },
+  changePassword (id, userData) {
+    return Axios.patch(RESOURCE_NAME.concat(`/passwordRecovery/${id}`), userData)
+  },
+  generatePasswordResetToken (email) {
+    return Axios.patch(RESOURCE_NAME.concat(`/recoveryTokenGeneration`), email)
   },
   update (id, data) {
     return Axios.put(`${RESOURCE_NAME}/${id}`, data)
   },
-
   delete (id) {
     return Axios.delete(`${RESOURCE_NAME}/${id}`)
   }
```

Comment:
update delete co? Quiz, użytkownik, pytanie, czy wszystko naraz?

Reply:
HTTP DELETE z założenia powinno usuwać cały resource na serwerze.
HTTP PUT powinno updateować cały resource na serwerze.
HTTP PATCH powinno modyfikować część resource na serwerze.
W tym wypadku jak ktoś będzie korzystał z tych funkcji to zrobi to tak:
UserService.delete(1) 
Więc myślę, że nazwa userservice wskazuje ze tym resource będzie user.